app/controllers/homes_controller.rb

```diff
@@ -16,12 +16,17 @@ def show
 
   # GET /homes/new
   def new
+    @agents = User.where('is_agent = true')
     @home = Home.new
   end
 
   # GET /homes/1/edit
   def edit
-    redirect_to homes_path unless @home.user_authorized?(current_user)
+    @agents = User.where('is_agent = true')
```

Comment:
Same as above.

Also go read about `scopes` to see how we could write this as:   `User.agents` where


`scope :agents, -> { where(is_agent: true) }`